apisix/admin/init.lua

```diff
@@ -354,7 +353,7 @@ function _M.init_worker()
 
     if local_conf.apisix.stream_proxy then
         local router_stream = require("apisix.stream.router.ip_port")
-        router_stream.stream_init_worker(filter)
+        router_stream.stream_init_worker()
```

Comment:
We need to export the filter and pass it into `stream_init_worker`, otherwise, the data structure may be different.